roles/ovirt-cluster-upgrade/library/ovirt_scheduling_policies_facts.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/python
```

Comment:
Let's make a note somewhere, that we should remove this module from the role once we bump dependency to Ansible 2.4.0